UI/GovernmentWnd.cpp

```diff
@@ -524,6 +571,20 @@ void PoliciesListBox::HideAllCategories(bool refresh_list) {
         Populate();
 }
 
+void PoliciesListBox::ResizePolicies(const GG::Pt& sz, const int pts) {
+    auto it = begin();
```

Comment:
didn't like / want to use something like my version?

Reply:
No, I tried, but didn't manage to access the list's items.